inference/serve.py

```diff
@@ -60,8 +71,9 @@ def main(argv=None):
     import argparse
     parser = argparse.ArgumentParser(description="Model Serve Script", add_help=False)
     parser.add_argument("--config_file", type=str, help="inference configuration file in YAML. If specified, all other arguments are ignored")
-    parser.add_argument("--model", default=None, type=str, help="model name or path")
-    parser.add_argument("--tokenizer", default=None, type=str, help="tokenizer name or path")
+    parser.add_argument("--model_id_or_path", default=None, type=str, help="model name or path")
+    parser.add_argument("--tokenizer_id_or_path", default=None, type=str, help="tokenizer name or path")
+    parser.add_argument("--model_to_serve", nargs='*', default=["gpt2"], type=str, help="Only used when config_file and model_id_or_path are both None, it needs to be a subset of the values of key 'name' in inference/models/*.yaml")
```

Comment:
models_to_serve? since it could be multiple models.

Reply:
We can use list arguments to specify multiple models.
python serve.py --model model1 --model model2
https://stackoverflow.com/questions/15753701/how-can-i-pass-a-list-as-a-command-line-argument-with-argparse

Reply:
> models_to_serve? since it could be multiple models.

yes, seems better to me. 

@KepingYan you also need to modify DOC for supporting multiple models.